config/pre-install/v0.16.0/dummy.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package preinstall is a placeholder that allows us to pull in config files
+// via go mod vendor.
+package preinstall
```

Comment:
what's the difference between "preinstall" and upgrade ? 

just wondering, since we have post-install, and upgrade already. and now pre-install ?

Just wondering

Reply:
I think upgrade might have been a poor name. I think it's cleaner to indicate which steps need to happen before upgrade (preinstall) and which have to happen after (postinstall). Anyways, that was my thinking :)

Reply:
no, I like

pre-install
post-install

I just did not really like upgrade :-)